Include/XA/kzla/kyum.tfl

```diff
@@ -1,789 +0,0 @@
-
-//***Taxiways***
-//taxiway

```

Comment:
KYUM has actually been renamed to KNYL. Any chance we could get this moved instead of deleted?

Reply:
Resolved by d0f219e